pkgs/build-support/build-bazel-package/default.nix

```
@@ -42,7 +42,7 @@ in stdenv.mkDerivation (fBuildAttrs // {
   inherit name bazelFlags bazelBuildFlags bazelFetchFlags bazelTarget;
 
   deps = stdenv.mkDerivation (fFetchAttrs // {
-    name = "${name}-deps";
+    name = "${name}-deps.tar.gz";
     inherit bazelFlags bazelBuildFlags bazelFetchFlags bazelTarget;
```

```suggestion
    inherit bazelFlags bazelBuildFlags bazelFetchFlags bazelTarget;
    
    impureEnvVars = lib.fetchers.proxyImpureEnvVars;
```
Since we're building a fixed output derivation fetching off the internet, the Bazel fetcher should have this like all the others do.